package.xml

```diff
@@ -18,6 +18,7 @@
 
   <build_depend>boost</build_depend>
   <build_depend>pluginlib</build_depend>
+  <build_depend>qtbase5-dev</build_depend>
```

Comment:
hmm that's a very good point. I guess we'll need to branch out here then, or am I an obvious alternative here ?